jobs/auctioneer/spec

```diff
@@ -88,7 +88,13 @@ properties:
   diego.auctioneer.locket.api_location:
     description: "Hostname and port of the Locket server. When set, the auctioneer attempts to claim a lock from the Locket API."
     default: locket.service.cf.internal:8891
-
+  diego.auctioneer.locket.client_keepalive_time: 
+    description: "Period in seconds after which the locket gRPC client sends keepalive ping requests to the locket server it is connected to."
+    default: 10
+  diego.auctioneer.locket.client_keepalive_timeout: 
+    description: "Timeout in seconds to receive a response to the keepalive ping. If a response is not received within this time, the locket client will reconnect to another server."
+    default: 20
+    
```

Comment:
This means it sends a keepalive every 10 seconds right? And that after 20 seconds it times out? 

If that is the case, can you change the timeout to be slightly more than 2 times larger than the keep alive? That way if the first keep alive is missed, the second one has a chance to get there in time. Maybe 22 seconds? 

Can you also add a template check (and test) for making sure the timeout is greater than the keepalive time?

Reply:
Yes, this makes sense. Im not sure where are the job templates being tested. In the bosh docs: https://bosh.io/docs/job-templates/  it is described that we need a bosh-template ruby gem. Currently we do not have something like this, should I create one for every job that we need to test ?